src/HeaderCell.tsx

```diff
@@ -24,7 +23,6 @@ export interface HeaderCellProps<R, SR> extends SharedHeaderRowProps<R, SR> {
 }
 
 export default function HeaderCell<R, SR>({
-  height,
```

Comment:
Unused prop. I wish we avoided doing `...props` where possible, as it can hide dead code.

Reply:
Let's remove any usage of `...props` after this pr